SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialDispatcher.h

```diff
@@ -28,13 +27,40 @@ class SPATIALGDK_API USpatialDispatcher : public UObject
 	GENERATED_BODY()
 
 public:
+	using FCallbackId = uint32;
+
 	void Init(USpatialNetDriver* NetDriver);
 	void ProcessOps(Worker_OpList* OpList);
 
+	// AddOpCallback returns a callback ID which is incremented on each callback that is registered.
+	// ComponentId must be in the range 1000 - 2000.
```

Comment:
Is this checked anywhere in code? I saw it in the docs, but couldn't see constants defining this range or validation when registering a callback or invoking callbacks.

Reply:
It's only checked at runtime, so users can currently register for ops that won't get forwarded back to them which is troll I agree, will add an assertion to add callbcaks

Reply:
oh I do already

Reply:
`check(SpatialConstants::MIN_EXTERNAL_SCHEMA_ID <= ComponentId && ComponentId <= SpatialConstants::MAX_EXTERNAL_SCHEMA_ID);`